gtime/metrics/metrics.py

```diff
@@ -114,3 +114,115 @@ def max_error(
 
     error = np.amax(np.absolute(np.subtract(y_true, y_pred)))
     return error
+
+def mse(
+    y_true: Union[pd.DataFrame, List, np.ndarray],
+    y_pred: Union[pd.DataFrame, List, np.ndarray],
+) -> float:
+    """Compute the mean squared error between two vectors.
+
+    Parameters
+    ----------
+    y_true : array-like, shape (length, 1), required
+        The first vector.
+
+    y_pred : array-like, shape (length, 1), required
+        The second vector.
+
+    Returns
+    -------
+    error : float
+        The mean squared error between the two vectors.
+
+    Examples
+    --------
+    >>> from gtime.metrics import mse
+    >>> y_true = [0, 1, 2, 3, 4, 5]
+    >>> y_pred = [1.1, 2.3, 0.4, 3.9, 3.1, 4.6]
+    >>> mse(y_true, y_pred)
+    1.20
+
+    """
+    y_true, y_pred = _convert_to_ndarray(y_true, y_pred)
+    _check_input(y_true, y_pred)
+    
+    sum_squared_error = sum((np.subtract(y_true, y_pred)) ** 2)
```

Comment:
I would change to:

`sum_squared_error = sum((y_true, y_pred) ** 2)`

Also below. The `np.subtract` is not too readable.